tools/testdata-scripts/src/bin/make-testdata.rs

```diff
@@ -47,25 +47,34 @@ fn main() {
         File::create(out_dir.join("testdata.postcard")).unwrap(),
```

Comment:
Nit: always include instructions for how to re-generate files, such as `complex_word-b4...` which you're adding to source control. Ideally write a generator task for cargo-make.

Question: Why did work_log.wasm change?

Suggestion: you should be able to get complex_word significantly smaller if you build it with an (empty) blob provider. Please do so if able.